### ISSUE TITLE
[FEATURE]: Can we also include WPM for measuring speed as that's most industry standard

### ISSUE THREAD

**Issue by romiljain5** [webdevcody/code-racer#690]
### Feature Summary

Can we also include WPM (words per minute) while calculating result of a user
This can really attract users as most website follows WPM for results instead of CPM

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by LukasOSmith** [comment]
I thought it was more appropriate because coding is much slower than typing words because of special characters. 

SO what do you think of a speed unit concept ? Then users can pick for themselves

**Comment by romiljain5** [comment]
Yes, but WPM is the standard way of measuring typing speed, Even in general whenever we speak about speed we generally consider it in WPM, So I thought it would give us an edge over others.
